rl_chain/slates_chain.py

```diff
@@ -55,31 +55,46 @@ def __init__(self, model: Optional[Any] = None, *args, **kwargs):
 
         self.model = model
 
-    def to_action_features(self, inputs: Dict[str, List[str]]):
+    def to_action_features(
+        self, inputs: Dict[str, List[str]], actions: Dict[str, Any]
+    ):
         def _str(embedding):
             return " ".join([f"{i}:{e}" for i, e in enumerate(embedding)])
 
-        named_actions = inputs.get("named_actions", None)
-        if named_actions is None:
-            raise ValueError("named_actions must be provided")
-
         action_features = [
             [
-                _str(self.model.encode(action.impl))
+                _str(self.model.encode(action.value))
                 if isinstance(action, base._Embed)
                 else action.replace(" ", "_")
                 for action in slot
             ]
-            for slot in named_actions.values()
+            for slot in actions.values()
         ]
 
         return action_features
 
     def to_vw_format(
-        self, inputs: Dict[str, Any], slates_label: Optional[Label] = None
+        self,
+        inputs: Dict[str, Any],
```

Comment:
I think we need inputs in to_vw_format (and predict) since I want that to be eventually an abstract class that can get implemented by the user and they might need inputs,

in to_action_features it is indeed not needed

Reply:
ok